openadapt/record.py

```diff
@@ -1189,6 +1358,12 @@ def record(
     )
     window_event_reader.start()
 
+    browser_event_reader = threading.Thread(
```

Comment:
I think we want to make this optional, similar to the Audio events.

In addition, we should increment `expected_starts` if browser events are enabled.

Reply:
Done here :bf5e43ea266b2f98eae7a77708a60a01d7aab9f3

Please double check if this is what we wanted!